rialto_airflow/google.py

```diff
@@ -8,8 +8,48 @@
 from airflow.providers.google.suite.hooks.sheets import GSheetsHook
 from airflow.providers.google.suite.hooks.drive import GoogleDriveHook
 from airflow.models import Variable
+import dotenv
+
+# Load environment variables
+dotenv.load_dotenv()
 
 gcp_conn_id = Variable.get("google_connection", "google_cloud_default")
+google_drive_id = Variable.get("google_drive_id", "some_google_drive_id")
```

Comment:
when running in stage/prod/localhost, this value should exist in .env or come from vault
for the tests, a bogus default is fine